### PR TITLE
Align standfirst bullets

### PR DESCRIPTION
**Pull request by HarryFischer**
## What does this change?
align standfirst bullets properly

### Before
Screenshot 2022-03-28 at 10 37 44

### After
Screenshot 2022-03-28 at 10 21 56

### CODE REVIEW

**Review by OllysCoding — APPROVED** [review]
+1

File: dotcom-rendering/src/web/components/Standfirst.tsx [hunk 1]
```diff
@@ -196,7 +248,8 @@ export const Standfirst = ({ format, standfirst }: Props) => {
 	return (
 		<div
 			css={[
-				nestedStyles(palette),
+				nestedStyles(format, palette),
+				// standfirstBulletStyles(format),
```
Comment (mxdvl) [inline dotcom-rendering/src/web/components/Standfirst.tsx:252]:
```suggestion
```

**Comment by mxdvl** [comment]
It’s hard to understand the differences between the two parts of the switch statement. What’s shared, and whats different?

File: dotcom-rendering/src/web/components/Standfirst.tsx [hunk 2]
```diff
@@ -196,7 +199,7 @@ export const Standfirst = ({ format, standfirst }: Props) => {
 	return (
 		<div
 			css={[
-				nestedStyles(palette),
+				nestedStyles(format, palette),
 				standfirstStyles(format, palette),
```
Comment (mxdvl) [inline dotcom-rendering/src/web/components/Standfirst.tsx:203]: For a subsequent PR, but worth noting that if you pass `format`, you can omit `palette`, which can be inferred via `decidePalette(format)`!

**Review by mxdvl — APPROVED** [review]

**Pull request merged**